### PR TITLE
Add Expert mode to IDP creation

### PR DESCRIPTION
**Pull request by vihanga-liyanage**
Refers: #354

idp-expert-mode

### CODE REVIEW

File: apps/admin-portal/src/components/identity-providers/meta/templates.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { IdentityProviderTemplateListItemInterface } from "../../../models";
```
Comment (brionmario) [inline apps/admin-portal/src/components/identity-providers/meta/templates.ts:1]: Shall we add the import after the copyright header.

File: apps/admin-portal/src/pages/identity-provider-template.tsx [hunk 2]
```diff
@@ -38,8 +38,9 @@ import { addAlert } from "@wso2is/core/store";
 import { AlertLevels } from "@wso2is/core/models";
 import { AppState } from "../store";
 import { setAvailableAuthenticatorsMeta } from "../store/actions/identity-provider";
-import { SupportedServicesInterface } from "../models/identity-provider";
+import { SupportedServicesInterface } from "../models";
 import { IdPCapabilityIcons } from "../configs";
+import { ExpertModeTemplate } from "../components/identity-providers/meta/templates";
```
Comment (brionmario) [inline apps/admin-portal/src/pages/identity-provider-template.tsx:43]: Shall we export this form `components` index.

File: apps/admin-portal/src/components/identity-providers/meta/templates.ts (hunk 1 above)
Comment (vihanga-liyanage) [reply apps/admin-portal/src/components/identity-providers/meta/templates.ts]: removed

File: apps/admin-portal/src/pages/identity-provider-template.tsx (hunk 2 above)
Comment (vihanga-liyanage) [reply apps/admin-portal/src/pages/identity-provider-template.tsx]: How about now?

**Pull request merged**